css/style.css

```diff
@@ -0,0 +1,49 @@
+
+
+body{
+
+    background-color: #4044ca;
```

Comment:
no need for overflow property

Reply:
It is needed brother .. for the screen to be as height of mobile